### ISSUE TITLE
The user/account Meticulus doesn't exist no more

### ISSUE THREAD

**Issue by Umeaboy** [oguzbakir/android_device_samsung_golden#1]
Is there any other updated device tree for Golden that I can use to build at least LOS 15.1?

**Comment by oguzbakir** [comment]
I can't really remember but I got successful boot with this tree using this manifest: https://github.com/oguzbakir/local_manifests_golden

Nothing works except booting, screen and touchscreen. If you want to build yourself, you have to apply patches in "android_device_samsung_golden/patches" folder using patches.sh.

**Comment by Umeaboy** [comment]
There's no local manifest for Lineage 16.0 if one wants to use that. Can you fix it?

**Comment by oguzbakir** [comment]
Yes, there is no local manifest for Lineage 16 because I couldn't fix build errors on this tree but Lineage 15.1 builds fine(If i remember correctly).

**Comment by Umeaboy** [comment]
What kernel do I use for lineage-15.1 then?
Your kernel is not for lineage-15.1 AFAICS.

**Comment by oguzbakir** [comment]
https://github.com/oguzbakir/local_manifests_golden/blob/lineage-15.1/local_manifest.xml#L9

**Comment by Umeaboy** [comment]
And the vendor blobs for 15.1?
